### PR TITLE
zulu-openjdk: Add armv8 support

### PR DESCRIPTION
**Pull request by uilianries**
fixes https://github.com/conan-io/conan-center-index/issues/10519

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/zulu-openjdk/all/conanfile.py [hunk 1]
```diff
@@ -24,8 +24,8 @@ def _jni_folder(self):
 
     def configure(self):
         if Version(self.version) < Version("11.0.12"):
-            if self.settings.arch != "x86_64":
-                raise ConanInvalidConfiguration("Unsupported Architecture.  This package currently only supports x86_64.")
+            if self.settings.arch not in ["x86_64", "armv8"]:
+                raise ConanInvalidConfiguration("Unsupported Architecture. This package currently only supports x86_64.")
```
Comment (hannesa2) [inline recipes/zulu-openjdk/all/conanfile.py:28]: I'm not sure, if the error message is the right

Comment (uilianries) [reply recipes/zulu-openjdk/all/conanfile.py:28]: Nope, I'll update. Thank for reviewing!

**Comment by uilianries** [comment]
I have been poisoned by my own remedy! Need to fix sha256 hook

**Comment by uilianries** [comment]
Blocked by https://github.com/conan-io/hooks/pull/410/

**Review by madebr — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Comment by uilianries** [comment]
Everything good now, the hooks has been fixed.

**Pull request merged**